python/api/src/araali/__main__.py

```diff
@@ -510,6 +511,52 @@ def fw_config(args):
     knobs = api.API().get_fw_config(zone=args.zone, tenant=args.tenant)
     print(yaml.dump(knobs))
 
+def quickstart(args):
+    if args.quickstart_subparser_name == "vm":
+        success, result = _aws.cf_launch_fortified_vm(stack_name="araali-quickstart-vm-stack", email=args.email, key_pair=args.key)
+        if not success:
+            print("Quickstart setup failed")
+            return
+        return result
+    elif args.quickstart_subparser_name == "eks":
+        EKS_STACK_NAME = "araali-quickstart-stack"
```

Comment:
cant use a static name. What if they run quickstart twice for two different VMs? Append a datetime>